### PR TITLE
VDB 1676 Create Base Images for Building and Running

### PR DESCRIPTION
**Pull request by paytonrules**
I'm not sure how to test the "deploy" script with actually merging to main.

### CODE REVIEW

File: dockerfiles/builder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Base image for building the docker containers for building vulcanize projects.
+# Meant to be both an optimization of the build, and to remove duplication of the
+# various setup steps in other Dockerfiles.
+FROM golang:1.15.2-buster as builder
+
+RUN apt-get update && \
+    apt-get install -y \
+    make git g++ linux-headers-amd64
+
+ENV GO111MODULE on
```
Comment (rmulhol) [inline dockerfiles/builder/Dockerfile:10]: Is this necessary?

**Review by rmulhol — APPROVED** [review]

File: dockerfiles/builder/Dockerfile (hunk 1 above)
Comment (rmulhol) [reply dockerfiles/builder/Dockerfile:10]: To clarify - I would think it's on by default now that we're at Go 1.15

Comment (rmulhol) [reply dockerfiles/builder/Dockerfile:10]: Welp looks like I'm wrong about that (defaults to `auto` which turns on if there's a `go.mod` in the directory). Though I do wonder if we could successfully install `goose` how we want without flipping on this env var (assuming it's not propagating to images built on top of this) 🤔

Comment (paytonrules) [reply dockerfiles/builder/Dockerfile:10]: I trust nothing with go modules, too traumatized. Also I figured it couldn't hurt :)

**Pull request merged**